docs/50_hub.md

```diff
@@ -0,0 +1,26 @@
+# Using the DANDI Hub
+
+[DANDI Hub](http://hub.dandiarchive.org) is a [JupyterHub](https://jupyterhub.readthedocs.io) instance in the cloud to interact with the data stored in DANDI, and is free to use for exploratory analysis of data on DANDI.
+Note that DANDI Hub is not intended for significant computation, but provides a place to introspect Dandisets and to perform some analysis and visualization of data.
+
+## Registration
+
+To use the [DANDI Hub](http://hub.dandiarchive.org), you must first register for an account using the [DANDI website](http://dandiarchive.org).
+
+## Choosing a server option
+
+When you start up the DANDI Hub, you will be asked to select across a number of server options.
+For basic exploration, Tiny or Base would most likely be appropriate.
+The DANDI Hub also currently offers Medium and Large options, which have more available memory and compute power.
+The "T4 GPU inference" server comes with an associated T4 GPU, and is intended to be used for applications that require GPU for inference.
+We request that users of this server be considerate of their usage of the DANDI Hub as a free community resource.
+Training large deep neural networks is not appropriate.
+A "Base (MATLAB)" server is also available, which provides a MATLAB cloud installation but you would be required to provide your own license.
+
+## Example notebooks
+
+The best way to share analyses on DANDI data is through the DANDI example notebooks.
```

Comment:
May be expand to

```suggestion
The best way to get started with existing data analyses or share your analyses on DANDI data is through the DANDI example notebooks.
```

Reply:
I would argue the best way to get started would be to start a notebook on the DANDI Hub, but I don't think users should use the "example notebooks" repo until they are ready to share them, so I prefer the text as it currently is.